andela/bp-esa-frontend
# #166726930 fix open profile icon functionality

#### What does this PR do?
- Open fellows AIS profile when profile icon button is clicked
- Show a tooltip when the mouse hovers on the button
#### Description of Task to be completed?

- The open profile icon button at the top right corner of the details modal should open the related developers profile on AIS in another tab when clicked

#### How should this be manually tested?
- Clone and set up the project using the instruction in the README.
- $ git checkout to `ch-open-ais-profile-166726930`
- Use $ `npm start to start` the application
- Visit http://localhost:3000.
- On the report page, click the information icon on the table view or automation card. A modal will pop up on the window. Locate open profile icon on the top right of the modal and click it. It should open a new AIS page with showing fellows profile.

#### Any background context you want to provide?
- N/A

#### What are the relevant pivotal tracker stories?

[166726930](https://www.pivotaltracker.com/story/show/166726930)
#### Postman Documentation

- N/a
#### Screenshots (If applicable)
![image](https://user-images.githubusercontent.com/17355400/59597154-df5e3a00-9101-11e9-9226-b4094628c8b0.png)

<details>
<summary>Details</summary>



</details>
